manifests/0000_26_cloud-controller-manager-operator_11_deployment.yaml

```diff
@@ -22,7 +22,7 @@ spec:
       serviceAccountName: cluster-cloud-controller-manager
       containers:
       - name: cluster-cloud-controller-manager
-        image: registry.svc.ci.openshift.org/openshift:cluster-cloud-controller-manager
+        image: registry.svc.ci.openshift.org/openshift:cluster-cloud-controller-manager-operator
```

Comment:
Shouldn't the image come from quay.io?

Reply:
No, this tag should be substituted with real image hash coming image-registry during image injection. The image is build for us by CI, and the operator is not intended to run outside openshift cluster.